confd/service/service.go

```diff
@@ -4,9 +4,8 @@ import (
 	"encoding/json"
 	"fmt"
 	"github.com/cloudogu/ces-confd/confd"
-	configRegistry "github.com/cloudogu/ces-confd/confd/registry"
-	"github.com/coreos/etcd/client"
 	"github.com/pkg/errors"
+	"go.etcd.io/etcd/client/v2"
```

Comment:
TBH I was shock-greeted by `go.etcd.io/etcd/client` package docs with a CVE-o-rama. The version you picked in `go.mod` looks cool though. :sweat_smile: There is a sliiiightly newer version out since 3 days but never mind. If it works it works.